boards/DIM/Test/Src/Test_SevenSegDisplays.cpp

```diff
@@ -6,46 +6,8 @@ DEFINE_FAKE_VOID_FUNC(set_left_hex_digit, struct SevenSegHexDigit);
 
 class SevenSegDisplaysTest : public DimTest
 {
```

Comment:
Hmm, I would think that this class would have all the seven seg setup and then the `Dim` test should extend this class. Otherwise this class gets everything in the dim test class, which doesn't really make sense with regards to making sure each class only gets what it needs and nothing more.

Reply:
whups i think you're totally right. I think this means StateMachineTeste will just inherit from multiple classes in the future?

```
class DimTest : public SevenSegDisplaysTest, public LedTest, public CanTest
```

I think i just forgot multiple inheritance was a thing ...

Reply:
Correct, and lmao.